samples/fabric/fabric-cli/.env.template

```diff
@@ -5,4 +5,5 @@ LOCAL=true
 DEFAULT_APPLICATION_CHAINCODE=simplestate
 DEFAULT_APPLICATION_FUNC=Create
 CONFIG_PATH=PATH-TO-CONFIG-FILE
+REMOTE_CONFIG_PATH=PATH-TO-REMOTE-CONFIG-FILE
```

Comment:
Move this line below the next to make it consistent with the instructions in the docs.

Reply:
Changed in docs, I remember maintaining this order somewhere with in fabric cli code as well.